tests/test_data/projects/integration_external/app/README.md

```diff
@@ -0,0 +1,4 @@
+# README
+
+This directory contains an extremely simple application that is used for
```

Comment:
nit: extremely simple is subjective, and might change in the future. How about something like "a native application for integ testing purposes", or something equally vague w.r.t scope?